modules/olm-operatorgroups-target-namespace.adoc

```diff
@@ -17,11 +17,15 @@ metadata:
   namespace: my-namespace
   spec:
     selector:
-      matchLabels:
-        cool.io/prod: "true"
+      cool.io/prod: "true"
 ----
 
-You can also explicitly name the target namespaces using the
+[NOTE]
+====
+Use of `matchLabels` with the `spec.selector` field is not recommended.
```

Comment:
The use of a label selector is what is not recommended. We should a text that explains why:

`The use of more than one target namespace in the OperatorGroup is not recommended as the support will likely be removed in the future`/